central/debug/service/service.go

```diff
@@ -614,6 +614,8 @@ func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseW
 	if err := zipWriter.Close(); err != nil {
 		log.Error(err)
 	}
+
+	w.WriteHeader(http.StatusOK)
```

Comment:
why do we need to write it explicitly here if the first `w.Write` call will set this header implicitly?

Reply:
Yup we don't need to do it but I added it to not change unit tests.